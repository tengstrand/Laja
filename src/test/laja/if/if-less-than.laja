#write "target/test-output/if-less-than.out"
  #set (x = 1)
  #if (x < 2)
x is less than 2
  #end
  #if (x < 1.1)
x is less than 1.1
  #end
  #if (x < "2")
x is less than "2"
  #end
  #if (x < 1)
x is less than 1
  #else
x is not less than 1
  #end
#end