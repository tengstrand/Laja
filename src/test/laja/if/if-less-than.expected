x is less than 2
x is less than 1.1
x is less than "2"
x is not less than 1
